docs/tfuser/readme.md

```diff
@@ -15,113 +15,175 @@ It has 3 main commands:
 Here is a full example how you would use tfuser to create a network schema.
 
 1. create a network from scratch
-`tfuser generate --schema network.json network create --node {nodeID}`
+`tfuser generate network create --name {NETWORK_NAME} --cidr ${CIDR}`
 
-the `--node` flag point to the node ID of the exit node we have chosen for the network.  
-the value of the `--schema` flag is a path where the provisoin schema will be written by tfuser. If not specified the schema will be printed to stdout.
+`NETWORK_NAME`: Arbitrary name of your network.
+`CIDR`: The prefix which will be used by the network resources in the network.
 
-2. add another node to the network:
-`tfuser generate --schema network.json network add-node --node {nodeID}`
+The schema will be printed on stdout, and should be saved for further actions.
 
-the `--node` flag point to the node ID we want to add to the network
+2. add another node to the network:
+`tfuser generate --schema ${SCHEMA_NAME} network add-node --node {NODE_ID} --subnet ${SUBNET} --port ${WGPORT}`
+
+`SCHEMA_NAME`: The file name of in which the current schema is stored. The new node
+	will be added to the schema, and the new schema version will be saved in the file.
+`NODE_ID`: The ID of the node to add in the reservation.
+`SUBNET`: The subnet to assign to the network resource on this node. All workloads
+	in this network resource will receive an IP from this subnet. The subnet must be
+	part of the `CIDR` provided when the network is created.
+`WGPORT`: Optional wireguard listening port on the host. If this is not specified, one will be
+	generated automatically. It is the responsibility of the caller to ensure this port
+	is not already in use.
 
 The result of these two command should looks like:
 ```json
 {
   "id": "",
-  "user_id": "E6Zb6cLHYuuXm1KSMeYc1DYNKfknGQqFVKS7tMc3mBvJ",
+  "user_id": "",
```

Comment:
Is this schema correct ? I count 4 network resource while the documentation only execute `add-node` command once.